### PR TITLE
Event class implementation A. Hiblov

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/entity/Event.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package entity;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by kowalskiy on 15.03.17.
+ */
```
Comment (dinamio) [inline src/main/java/entity/Event.java:8]: Two suggestions.
1. Participants must be List of Bands, not from Artist
2. address must be venue

**Review by dinamio — CHANGES_REQUESTED** [review]
Please, see comments below

File: src/main/java/entity/Event.java (hunk 1 above)
Comment (ghost) [reply src/main/java/entity/Event.java:8]: Внес изменения

**Review by dinamio — APPROVED** [review]
Looks good to me

**Pull request merged**